### ISSUE TITLE
Incompatible with perl 5.26 ~

### ISSUE THREAD

**Issue by aero** [melezhik/sparrow#26]
Hi,

Your
M/ME/MELEZHIK/Sparrow-0.3.2.tar.gz
M/ME/MELEZHIK/Outthentic-0.4.0.tar.gz
are now not compatible with perl 5.26 and above.

Please see http://blogs.perl.org/users/todd_rinaldo/2016/11/how-removing-from-inc-is-about-to-break-cpan.html

```
$ cpan Sparrow
Reading '/Users/user/.cpan/Metadata'
  Database was generated on Thu, 01 Nov 2018 00:29:03 GMT
Running install for module 'Sparrow'
CPAN: Digest::SHA loaded ok (v6.02)
CPAN: Compress::Zlib loaded ok (v2.081)
Checksum for /Users/user/.cpan/sources/authors/id/M/ME/MELEZHIK/Sparrow-0.3.2.tar.gz ok
CPAN: YAML loaded ok (v1.26)
CPAN: CPAN::Meta::Requirements loaded ok (v2.140)
CPAN: Parse::CPAN::Meta loaded ok (v2.150010)
CPAN: CPAN::Meta loaded ok (v2.150010)
CPAN: Module::CoreList loaded ok (v5.20181020)
Configuring M/ME/MELEZHIK/Sparrow-0.3.2.tar.gz with Makefile.PL
CPAN: CPAN::Reporter loaded ok (v1.2018)
Can't locate inc/Module/Install.pm in @INC (you may need to install the inc::Module::Install module) (@INC contains: ...... ) at Makefile.PL line 1.
BEGIN failed--compilation aborted at Makefile.PL line 1.
(/Users/user/perl5/perlbrew/perls/perl-5.26/bin/perl Makefile.PL exited with 512)
```

**Comment by melezhik** [comment]
I guess `use lib ".";` in the top of Makefile.PL should fix it .

**Comment by melezhik** [comment]
see also https://github.com/melezhik/outthentic/issues/11

**Comment by melezhik** [comment]
fixed in version 0.3.3 , please check
